Gopkg.toml

```diff
@@ -29,7 +29,7 @@ version="v1.4.7"
 
 [[constraint]]
   name = "github.com/knative/serving"
-  version = "v0.5.0"
+  revision = "a44e3578ce27cc1455b3e1c7f74b09b82dda57f5"
```

Comment:
can you leave a comment on which knative major version this is?

Reply:
I'm not sure which major version this is as its master? They are on v0.5.2 but not sure if all of this will be released in v0.5.3 as it includes work towards v1beta1.

Reply:
I think we should pin to the release v0.5.2, v1beta1 is not ready yet.